src/example-types/esTwoLevelAggregation.js

```diff
@@ -72,7 +75,17 @@ module.exports = {
                 key: bucket.key || key,
                 doc_count: bucket.doc_count,
               },
-              bucket.twoLevelAgg
+              bucket.twoLevelAgg ||
+                _.find(
+                  value =>
+                    !F.cascade(['value', 'values', 'value_count'], value) &&
+                    _.isObject(value),
+                  bucket
+                ) ||
+                _.flow(
+                  _.mapKeys(_.replace('value_', '')),
```

Comment:
I'm unclear why we're special casing `value_` here. If you do an include with `value_count`, you're going to pass up `value_count` explicitly. We're not making `count` magically work as a metric agg under include (even though it is the name of the property on the stats agg response), so it'd be weird to say `include: ['value_count']` and return `context: { count }`. This would only make sense if we made it so you could do `include: ['count']` (but I don't think we need to bother with that)

Reply:
Spark breaks allover the place when we don't do this so is kind of backwards compatability thing.